MAUI/Cartesian-Charts/Errorbar-Chart.md

```diff
@@ -0,0 +1,762 @@
+        XBindingPath = "ID",
+        YBindingPath = "Coal",
+        PointWidth = 20,
+        PointHeight = 20
+    };
+
+    ErrorBarSeries errorBar = new ErrorBarSeries()
+    {
+        ItemsSource = new ViewModel().EnergyProductions,
+        XBindingPath = "ID",
+        YBindingPath = "Coal",
+        HorizontalErrorValue = 0.5,
+        VerticalErrorValue = 50
+    };
+
+    chart.Series.Add(series);
+    chart.Series.Add(errorBar);
+
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Error Bar Series in MAUI Chart](Chart-types_images/BasicRenderingErrorBar.png)
+
+## Mode 
+This [Mode]() property defines whether to identify a horizontal error or vertical error. By default, the Mode value is [Both](), which will display both horizontal and vertical error values.
+
+### Horizontal
+
+To view horizontal error value, you can set the Mode as Horizontal as shown in the following code example.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:ErrorBarSeries ItemsSource="{Binding EnergyProductions}"
+                          XBindingPath="ID"
+                          YBindingPath="Coal"
+                          VerticalErrorValue="50"
+                          HorizontalErrorValue="0.5"
+                          Mode="Horizontal">
+    </chart:ErrorBarSeries>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    ErrorBarSeries errorBar = new ErrorBarSeries()
+    {
+        ItemsSource = new ViewModel().EnergyProductions,
+        XBindingPath = "ID",
+        YBindingPath = "Coal",
+        HorizontalErrorValue = 0.5,
+        VerticalErrorValue = 50,
+        Mode= Horizontal
+    };
+    chart.Series.Add(series);
+    chart.Series.Add(errorBar);
+
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Horizontal Mode Support in Error Bar Series](Chart-types_images/ModeHorizontal.png)
+
+
+### Vertical
+
+To view vertical error value, you can set the Mode as Vertical, as shown in the below code example.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:ErrorBarSeries ItemsSource="{Binding EnergyProductions}"
+                          XBindingPath="ID"
+                          YBindingPath="Coal"
+                          VerticalErrorValue="50"
+                          HorizontalErrorValue="0.5"
+                          Mode="Vertical">              
+    </chart:ErrorBarSeries>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    ErrorBarSeries errorBar = new ErrorBarSeries()
+    {
+        ItemsSource = new ViewModel().EnergyProductions,
+        XBindingPath = "ID",
+        YBindingPath = "Coal",
+        HorizontalErrorValue = 0.5,
+        VerticalErrorValue = 50,
+        Mode= Vertical
+    };
+    chart.Series.Add(series);
+    chart.Series.Add(errorBar);
+
+
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Vertical Mode Support in Error Bar Series](Chart-types_images/ModeVertical.png)
+
+
+## Direction 
+
+[ErrorBar]() series allows you to view the horizontal and vertical error values in both positive and negative directions.
```

Comment:
The HorizontalDirection() and VerticalDirection() specifies whether to show positive, negative, or both directions of error values.

Reply:
ErrorBarDirection() contains below values:
*`Both` -  It indicates the actual data point value along with specific amount of positive and negative error values.

*`Plus` -  It indicates the actual data point value along with specific amount of positive error value.

*`Minus`-  It indicates the actual data point value along with specific amount of negative error value.

Reply:
I replaced the direction part